lib/DelegatedModule.js

```diff
@@ -87,6 +87,11 @@ class DelegatedModule extends Module {
 	size() {
 		return 42;
 	}
+
+	updateHash(hash) {
+		hash.update(this.identifier());
```

Comment:
Could you update the hash with `this.type` and `this.request` instead of `this.identifier()`?

Reply:
Sure. I JSON.stringified it since `this.request` may be a number it will cause TypeError when passed to `hash.update`.